docs/sdks/server-sdks/node.md

```diff
@@ -49,8 +49,8 @@ import { IAssignmentLogger, init } from "@eppo/node-server-sdk";
 // Define logAssignment so that it logs events
 const assignmentLogger: IAssignmentLogger = {
   logAssignment(assignment) {
-		console.log(assignment)
-  },
+	console.log(assignment);
+  }
```

Comment:
This file was inconsistent with trailing commas, so I decided to align on my personal preference of no trailing commas. 😅